### PR TITLE
app-shells/starship: add 1.11.0

### PR DESCRIPTION
**Pull request by ixti**
* Min rust version bumped to 1.60.0
* New licence requirement added: Unicode-DFS-2016
 
Signed-off-by: Alexey Zapparov <alexey@zapparov.com>

### CODE REVIEW

**Comment by ixti** [comment]
cc: @bowlofeggs

File: app-shells/starship/starship-1.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,430 @@
+# Copyright 2022 Gentoo Authors
```
Comment (bowlofeggs) [inline app-shells/starship/starship-1.11.0.ebuild:1]:
```suggestion
# Copyright 2020-2022 Gentoo Authors
```

File: app-shells/starship/starship-1.11.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+	winres-0.1.12
+	yaml-rust-0.4.5
+	zbus-2.3.2
+	zbus_macros-2.3.2
+	zbus_names-2.2.0
+	zvariant-3.6.0
+	zvariant_derive-3.6.0
+"
+
+inherit cargo
```
Comment (bowlofeggs) [inline app-shells/starship/starship-1.11.0.ebuild:385]: `flag-o-matic` is needed for `filter-lto`:

```suggestion
inherit cargo flag-o-matic
```

File: app-shells/starship/starship-1.11.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+	https://github.com/starship/starship/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris)
+"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC0-1.0 ISC MIT MPL-2.0 Unicode-DFS-2016 Unlicense WTFPL-2 ZLIB"
+
+SLOT="0"
+KEYWORDS="amd64"
+
+BDEPEND=">=virtual/rust-1.64"
```
Comment (bowlofeggs) [inline app-shells/starship/starship-1.11.0.ebuild:399]: Shouldn't this be 1.60? I see that here: https://github.com/starship/starship/blob/v1.11.0/Cargo.toml#L26

```suggestion
BDEPEND=">=virtual/rust-1.60"
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

File: app-shells/starship/starship-1.11.0.ebuild (hunk 3 above)
Comment (ixti) [reply app-shells/starship/starship-1.11.0.ebuild:399]: Indeed. By some reason I used min version from the master branch of the starship. o_O

**Comment by ixti** [comment]
@bowlofeggs thanks for the review. Updated the PR with corrections.

File: app-shells/starship/starship-1.11.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+HOMEPAGE="https://starship.rs/"
+SRC_URI="
+	https://github.com/starship/starship/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris)
+"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC0-1.0 ISC MIT MPL-2.0 Unicode-DFS-2016 Unlicense WTFPL-2 ZLIB"
+
+SLOT="0"
+KEYWORDS="amd64"
```
Comment (bowlofeggs) [inline app-shells/starship/starship-1.11.0.ebuild:397]: Sorry, I missed one other thing during the last review ☹

```suggestion
KEYWORDS="~amd64"
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
LGTM, except for the `KEYWORDS`.

File: app-shells/starship/starship-1.11.0.ebuild (hunk 4 above)
Comment (ixti) [reply app-shells/starship/starship-1.11.0.ebuild:397]: :facepalm: I should be more attentive when I copy parts from the previous version ebuild...

Comment (ixti) [reply app-shells/starship/starship-1.11.0.ebuild]: Fixed

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
I don't have commit privileges, so we'll need to wait for someone else to approve and merge. Thanks for the patch!

**Comment by thesamesam** [comment]
thanks folks

File: app-shells/starship/starship-1.11.0.ebuild (hunk 4 above)
Comment (juippis) [reply app-shells/starship/starship-1.11.0.ebuild]: Try to develop a habit for calling **ekeyword ~all new.ebuild** anytime you bump an ebuild :)

Comment (ixti) [reply app-shells/starship/starship-1.11.0.ebuild]: @juippis Thanks for the tip (didn't know of the command). Will add it to my checklist! :D

**Pull request closed without merging**